.github/workflows/goreleaser.yml

```diff
@@ -43,3 +45,22 @@ jobs:
         KO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         USER: ${{ github.actor }}
         EMAIL: ${{ github.actor}}@users.noreply.github.com
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v4
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Login to registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Package and push chart
+      run: |
+        chartVersion=$(echo ${{ github.ref_name }} | cut -c 2-)
+        sed -i 's@OTP_IMAGE_PLACEHOLDER@${{ env.REGISTRY }}/${{ env.CONTROLLER_IMAGE }}:${{ github.ref_name }}@g' ${{ env.CHART_PATH }}/templates/controller.yaml
```

Comment:
I wrote in the controller yaml file about possible typo.. atm you didn't put `OTP_IMAGE_PLACEHOLDER` in the deployment template.
if you fix the other fix this should probably be fine.

Reply:
yes, I forgot to uncomment the controller yaml after testing.